src/jsinteractive.c

```diff
@@ -185,7 +185,7 @@ NO_INLINE void jsiConsolePrintString(const char *str) {
   }
 }
 
-#ifdef FLASH_STR
+#ifdef USE_FLASH_MEMORY
```

Comment:
As described in jsutils.h I had to switch from using defined(FLASH_STR) as an implicit indication that we're doing flashy stuff with constant strings to using an explicit #define just for that.